.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: node_js
-node_js:
-  - lts/*
-cache:
-  directories:
-    - "~/.npm"
+node_js: 12
 notifications:
   email:
     recipients:
```

Comment:
while you're in here: should we change the recipients to `masters-devs` (or similar)?